examples/meta/generator/targets/doc.js

```diff
@@ -0,0 +1,239 @@
+/** Target files specify rules for translation from meta-language to a target
```

Comment:
JavaScript. JSON does not support comments, so syntax highlighters make this hard to read. I could also change to markdown and wrap this in a code segment?